Sources/BraintreeThreeDSecure/BTThreeDSecureError.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+
+enum BTThreeDSecureError: Int, Error, CustomNSError, LocalizedError {
+
+    /// Unknown error
+    case unknown
+
+    /// 3D Secure failed during the backend card lookup phase; please retry
+    case failedLookup
+
+    /// 3D Secure failed during the user-facing authentication phase; please retry
+    case failedAuthentication
+
+    /// 3D Secure was not configured correctly
+    case configuration
+
+    /// A body was not returned from the API during the request.
+    case noBodyReturned
+
+    static var errorDomain: String {
+        "com.braintreepayments.BTThreeDSecureFlowErrorDomain"
+    }
```

Comment:
```suggestion
    static let errorDomain: String = "com.braintreepayments.BTThreeDSecureFlowErrorDomain"
```
🧶 : does this compile as a plain constant?